src/modules/app/scanner.js

```diff
@@ -19,7 +19,7 @@ const keyPressTimer$ = keyPressReducer$.debounce(() => Observable.timer(50))
     return {...state, buffer: '', code: buffer, uuid};
```

Comment:
This is really a tried and tested UUID regex. The time taken to write the test is not worth the cause. Since we dont validate UUID any other place. when we introduce UUID AS ID for object and node then its another case